sources/au/victoria.json

```diff
@@ -23,10 +23,21 @@
     "conform": {
         "file": "ll_gda94/sde_shape/whole/VIC/VMADD/layer/address.shp",
         "type": "shapefile",
-        "number": "NUM_ADD",
+        "id": "PFI",
+        "unit": {
+            "function": "format",
+            "fields": ["bunit_pre1", "bunit_id1", "bunit_suf1", "bunit_pre2", "bunit_id2", "bunit_suf2"],
+            "format": "$1$2$3-$4$5$6"
+        },
+        "number": {
+            "function": "format",
+            "fields": ["hse_pref1", "hse_num1", "hse_suf1", "hse_pref2", "hse_num2", "hse_suf2"],
+            "format": "$1$2$3-$4$5$6"
+        },
         "street": [
             "ROAD_NAME",
-            "ROAD_TYPE"
+            "ROAD_TYPE",
+            "RD_SUF"
```

Comment:
This was overlooked before, but should be there.

Reply:
no I think not

Reply:
Could you please explain why not? There is a field in the source data ezi_address which is the complete formatted address which includes the RD_SUF, so if the ezi_address has it, then I think the OA address should.